Matlab/xdf/load_xdf.m

```diff
@@ -502,7 +503,7 @@
                 segments(r).t_begin = temp(k).time_stamps(range(1));
                 segments(r).t_end = temp(k).time_stamps(range(2));
                 segments(r).duration = segments(r).t_end - segments(r).t_begin;
-                segments(r).effective_srate = (segments(r).num_samples-1)/ segments(r).duration;
```

Comment:
This change here was applied to 1.11 in a 3rd party contribution without bumping up the version number, and was unfortunately not reflected in the 1.12 version. It's probably best to carry this line over into 1.12 in the repo.